### DISCUSSION TITLE
A little aside, what tools did you use to make your GIFs in the doc.

### DISCUSSION THREAD

**Discussion by yixian-huang** [lisonge/vite-plugin-monkey#109] [Q&A] [answered]
I think these GIFs are very graphic and accurate with shortcut key tips and progress bars.

Thank you.

**Comment by lisonge** [comment, marked as answer]
<https://github.com/NickeManarin/ScreenToGif>

if you use window [scoop](https://github.com/ScoopInstaller/Scoop)

```shell
scoop install screentogif
```
